examples/fluids/navierstokes.h

```diff
@@ -262,19 +262,35 @@ struct AdvectionContext_ {
 };
 #endif
 
+// Newtonian Ideal Gas
+#ifndef newtonian_context_struct
+#define newtonian_context_struct
+typedef struct NewtonianIdealGasContext_ *NewtonianIdealGasContext;
+struct NewtonianIdealGasContext_ {
+  CeedScalar lambda;
+  CeedScalar mu;
+  CeedScalar k;
+  CeedScalar cv;
+  CeedScalar cp;
+  CeedScalar g;
+  CeedScalar c_tau;
+  StabilizationType stabilization;
+};
+#endif
+
 // Struct that contains all enums and structs used for the physics of all problems
 struct Physics_private {
-  DCContext            dc_ctx;
-  EulerContext         euler_ctx;
-  AdvectionContext     advection_ctx;
-  WindType             wind_type;
-  BubbleType           bubble_type;
-  BubbleContinuityType bubble_continuity_type;
-  EulerTestType        euler_test;
-  StabilizationType    stab;
-  PetscBool            implicit;
-  PetscBool            has_curr_time;
-  PetscBool            has_neumann;
+  NewtonianIdealGasContext newt_ig_ctx;
```

Comment:
Can we change `newt_ig_ctx` to `newtonian_ig_ctx`? We have been trying to choose more descriptive variable names.

Reply:
I just pushed this, thanks.